pkg/operator/controller/extension/utils/extensions.go

```diff
@@ -0,0 +1,88 @@
+//  SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
+//
+//  SPDX-License-Identifier: Apache-2.0
+
+package utils
+
+import (
+	_ "embed"
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/util/json"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/apimachinery/pkg/util/strategicpatch"
+	"sigs.k8s.io/yaml"
+
+	operatorv1alpha1 "github.com/gardener/gardener/pkg/apis/operator/v1alpha1"
+	"github.com/gardener/gardener/pkg/utils"
+)
+
+var (
+	//go:embed extensions.yaml
+	extensionsYAML string
+	extensions     map[string]Extension
+)
+
+// Extension is the default specification for an `operator.gardener.cloud/v1alpha1.Extension` object.
+type Extension struct {
+	// Name is the name of the extension (without `gardener-extension-` prefix).
+	Name string `json:"name" yaml:"name"`
+	// Annotations are additional annotations that may apply to the extension and control behavior.
+	Annotations map[string]string `json:"annotations" yaml:"annotations"`
```

Comment:
Also add labels?

Reply:
Do we have a need for the labels ? The annotations are functionally important for https://github.com/gardener/gardener/blob/master/docs/extensions/conventions.md#how-to-enforce-a-pod-security-standard-for-extension-namespaces . I would expect that the labels should be added by the landscape operator and not really get defaulted by us. Unless there are some labels that affect functionality.

Reply:
Yes, I was just confused because the "pod security level" seems to have been become a field in the spec. In this light, we could drop annotations and not introduce labels at all?